src/main/java/com/couchbase/intellij/workbench/Log.java

```diff
@@ -24,8 +25,15 @@ public static boolean isDebug() {
 
     public static ConsoleView getLogger() {
         if (console == null) {
-            console = TextConsoleBuilderFactory.getInstance().createBuilder(
-                    ProjectManager.getInstance().getDefaultProject()).getConsole();
```

Comment:
IntelliJ has a recommended logger that is integrated with their error reporting system: `com.intellij.openapi.diagnostic.Logger`